devstudio/pom.xml

```diff
@@ -49,7 +49,8 @@ http://download.jboss.org/jbosstools/builds/staging/JBDSIS-aggregate-disc/all/re
     <PREFIX>devstudio-integration-stack</PREFIX>
     <TARGET_PLATFORM>luna</TARGET_PLATFORM>
     <VERSION>8.0.0.CR2</VERSION>
-    <IS_TP_VERSION>4.2.0.Final</IS_TP_VERSION>
+    <IS_TP_VERSION>4.2.1.Final</IS_TP_VERSION>
+    <IS_TP_EA_VERSION>4.2.1.EA</IS_TP_EA_VERSION>
```

Comment:
the version is still 4.2.1.Final IMO

it is the artifact that changes, not the version.